### PR TITLE
Update zh-cn.cfg

### PR DESCRIPTION
**Pull request by tinygrox**
For commit a3937d217c78af1cf46d3c29a7bf092e84cd6926
😄

### CODE REVIEW

File: RealFuels/Localization/zh-cn.cfg [hunk 1]
```diff
@@ -114,7 +118,7 @@ Localization
         #RF_FuelPump_FuelPump = 燃油泵
         #RF_FuelPump_PumpRate = 泵送速率
 
-        #RF_FuelTank_Default = 默认
+        #RF_FuelTank_Default = Default
```
Comment (siimav) [inline RealFuels/Localization/zh-cn.cfg:121]: Is this change intentional?

Comment (tinygrox) [reply RealFuels/Localization/zh-cn.cfg:121]: Yeah, can't say why. My intuition made me. 😬
